### PR TITLE
Create GitHub workflow trivy.yml for scanning of security vulnerabilities

### PR DESCRIPTION
**Pull request by cruizen**
GitHub action to automate scanning of vulnerabilities with Trivy

### CODE REVIEW

**Comment by cruizen** [comment]
The output from the first run https://github.com/platform9/luigi/pull/48/checks?check_run_id=11339400064 shows that there are 2 high vulns that need to be fixed.

File: .github/workflows/trivy.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: build
```
Comment (Rohitrajak1807) [inline .github/workflows/trivy.yml:6]: Changing this name would be better.

File: .github/workflows/trivy.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  schedule:
+    - cron: '42 17 * * 0'
+
+permissions:
+  contents: read
+
+jobs:
+  build:
```
Comment (Rohitrajak1807) [inline .github/workflows/trivy.yml:21]:
```suggestion
  build_image:
```

File: .github/workflows/trivy.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    permissions:
+      contents: read # for actions/checkout to fetch code
+      security-events: write # for github/codeql-action/upload-sarif to upload SARIF results
+      actions: read # only required for a private repository by github/codeql-action/upload-sarif to get the Action run status
+    name: Build
+    runs-on: "ubuntu-18.04"
```
Comment (Rohitrajak1807) [inline .github/workflows/trivy.yml:27]:
```suggestion
    runs-on: "ubuntu-latest"
```

**Review by Rohitrajak1807 — CHANGES_REQUESTED** [review]

File: .github/workflows/trivy.yml (hunk 1 above)
Comment (cruizen) [reply .github/workflows/trivy.yml]: Thanks, done

**Review by Rohitrajak1807 — APPROVED** [review]

**Review by mithilarun — APPROVED** [review]

**Pull request merged**